### PR TITLE
chore(release): updated changelogs for 0.1.0

### PR DESCRIPTION
**Pull request by fjnoyp**
*Issue #, if available:*

*Description of changes:*
Update changelogs for 0.1

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/amplify_auth_cognito/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+## 0.1 (2021-02-12)
+
+### Features
+
+- feat: sign in with apple (#378)
+
+### Bug Fixes
+
+- fix: force signOut when signedIn (#369)
```
Comment (haverchuck) [inline packages/amplify_auth_cognito/CHANGELOG.md:9]: include
`- fix(amplify_auth_cognito): deprecates globalSignOut; simple user check upon Android signIn`

**Review by jamesonwilliams — APPROVED** [review]

File: example/pubspec.yaml [hunk 2]
```diff
@@ -25,10 +25,10 @@ dependencies:
   flutter:
     sdk: flutter
   file_picker: ^1.8.0+1
-  amplify_flutter: ^0.0.2-dev.2
-  amplify_analytics_pinpoint: ^0.0.2-dev.2
-  amplify_auth_cognito: ^0.0.2-dev.2
-  amplify_storage_s3: ^0.0.2-dev.2
+  amplify_flutter: 0.1
```
Comment (haverchuck) [inline example/pubspec.yaml:28]: are you sure two digit version is valid?

File: example/pubspec.yaml [hunk 3]
```diff
@@ -25,10 +25,10 @@ dependencies:
...
+  amplify_analytics_pinpoint: 0.1
+  amplify_auth_cognito: 0.1
+  amplify_storage_s3: 0.1
```
Comment (Ashish-Nanda) [inline example/pubspec.yaml:31]: Can we make it `0.1.0` throughout the PR please?

**Comment by Amplifiyer** [comment]
the version field  of the package in the pubspec.yaml also needs to be updated.

**Review by Ashish-Nanda — APPROVED** [review]
LGTM

**Review by Ashish-Nanda — CHANGES_REQUESTED** [review]
Please update the version field of the `pubspec.yaml` to `0.1.0` as well for the packages.

**Review by Ashish-Nanda — APPROVED** [review]
LGTM

**Review by Amplifiyer — APPROVED** [review]

**Review by Ashish-Nanda — APPROVED** [review]
Looks good

**Pull request merged**